hotline/species/forms.py

```diff
@@ -0,0 +1,67 @@
+from django import forms
+from django.utils.safestring import mark_safe
+from elasticmodels.forms import SearchForm
+
+from .indexes import SpeciesIndex
+
+
+class SpeciesSearchForm(SearchForm):
+    """
+    This form handles searching for a species in the species list view.
+    """
+    q = None
+
+    querystring = forms.CharField(required=False, widget=forms.widgets.TextInput(attrs={
+        "placeholder": "name:Foobarius"
+    }), label=mark_safe("Search <a target='_blank' class='help' href='help'>[?]</a>"))
+
+    sort_by = forms.ChoiceField(choices=[
+        ("name", "Name"),
+        ("scientific_name", "Scientific Name"),
+        ("severity", "Severity"),
+        ("category", "Category"),
+        ("is_confidential", "Confidential"),
+    ], required=False)
+
+    order = forms.ChoiceField(choices=[
+        ("ascending", "Ascending"),
+        ("descending", "Descending"),
+    ], required=False, initial="ascending", widget=forms.widgets.RadioSelect)
+
+    def __init__(self, *args, user, **kwargs):
+        self.user = user
+        super().__init__(*args, index=SpeciesIndex, **kwargs)
+
+    def get_queryset(self):
```

Comment:
If you were to override the (undocumented) get_queryset method on your subclass of Index, you wouldn't have to override this method.